java-checks/src/test/files/checks/AllBranchesAreIdentical.java

```diff
@@ -46,6 +66,11 @@ void ifStatement() {
     if (true) f(); // Noncompliant
     else f();
 
+    if(b == 0) { // Compliant: this is the exception case as defined in RSPEC
```

Comment:
Comment is a bit dodgy : maybe explain WHY it is compliant (because it has no else, and this case is handled by another rule)